### ISSUE TITLE
Individual Organizations

### ISSUE THREAD

**Issue by natebrunette** [pulumi/service-requests#178]
First, I want to say that I understand why organizations are a team feature and I think it makes sense. I'm almost assuredly in a minority, but it would be useful for me to be able to add single user organizations to my account because I have multiple different projects that I manage alone. I can put them all in one account, but I feel like it would be nicer if I had an upgrade path moving from a single user to team by adding users to an organization.

**Comment by EvanBoyle** [comment]
Could you provide a little bit more detail about the problem you're facing? It is certainly possible for a user to be in multiple team organizations.

**Comment by natebrunette** [comment]
Sure, I'd like to be able to create organizations in an individual account so I can use one account to manage multiple different projects without doing something like prefixing a stack with the project name to keep them separate or creating multiple accounts. If your username could be your email address, it wouldn't be that big of a deal to create multiple accounts.

**Comment by EvanBoyle** [comment]
Why do you need the account to be the boundary here? Where does the project model fall short for you? It is common to have 1000+ projects in a single organization.

**Comment by natebrunette** [comment]
No, definitely not that many. Currently the only pain is that I have to prefix project names and it's not as easy to view the stack I care about. If it's a huge pain to implement, it's not the most important feature, but it would be a nice organizational tool. Also, as I mentioned before, it would be easier to convert projects you start yourself to paid projects just by adding members to your already existing organization. However, maybe there could be another option layer of grouping that would work for both organizations and individuals.
